src/App.js

```diff
@@ -1,24 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './style.css';
 
 function App() {
   return (
```

Comment:
- [ ] As the requirements say that also the App.js should be class-based.

![Capture web_2-12-2021_163952_gitlab com](https://user-images.githubusercontent.com/61562626/144453887-2a4d4841-58b6-48f4-a1a8-5c71e7b39f21.jpeg)